CodeBenders/Assets/Scripts/ControlSlingshotFunctions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using UnityEngine;
+
+/// <summary>
```

Comment:
@bzhu36 @bharatht96 also this component shoukl have all the functions you need. You can use this as a message receiver.